source/text/implementation/vss-strings-cursors-internals.adb

```diff
@@ -51,6 +51,34 @@ package body VSS.Strings.Cursors.Internals is
       end if;
    end First_Cursor_Access_Constant;
 
+   --------------
+   -- Is_Owner --
+   --------------
+
+   function Is_Owner
+     (Self  : VSS.Strings.Cursors.Abstract_Cursor'Class;
+      Owner : VSS.Strings.Virtual_String'Class) return Boolean is
+   begin
+      if Self in VSS.Strings.Cursors.Character_Cursor_Limited_Base'Class then
```

Comment:
Can we have a common base implementation type that provides `Owner` property/method to avoid an `if`-`elsif` chain?

Reply:
May be... It is quite complicated issue, I'll try to investigate it later.